app/Console/Kernel.php

```diff
@@ -34,6 +35,7 @@ protected function schedule(Schedule $schedule)
         $schedule->job(new ExpireOldUserVerificationTokensJob)->hourly();
         $schedule->job(new PruneEventPageUpdatesTable)->everyFifteenMinutes();
         $schedule->job(new PruneQueryserviceBatchesTable)->everyFifteenMinutes();
+        $schedule->job(new RequeuePendingBatchesJob)->everyFifteenMinutes();
```

Comment:
Alternatively we could also have one "cleanup" type of job that does everything around QueryService batches.